test/rpc_test.py

```diff
@@ -1083,20 +1083,21 @@ def test_set_rpc_timeout(self):
         rpc.join_rpc()
 
     @dist_init
+    @requires_process_group_agent("PROCESS_GROUP rpc backend specific test, skip")
```

Comment:
Can we cut a task to enable this for other backends?

Reply:
Done